python_flask/app/models/clients.py

```diff
@@ -16,20 +16,19 @@ class Client(db.Model):
     created_at = db.Column(db.DateTime)
     updated_at = db.Column(db.DateTime)
 
-    def __init__(self, name, address, city, state, zip, country, email,
-        contact, phone, created_at, updated_at):
-        self.name = name
-        self.email = email
-        self.address = address
-        self.city = city
-        self.state = state
-        self.zip = zip
-        self.country = country
-        self.email = email
-        self.contact = contact
-        self.phone = phone
-        self.created_at = created_at.utcnow()
-        self.updated_at = updated_at.utcnow()
+    def __init__(self, attribute):
```

Comment:
Thinking about this some more, I wonder if inheriting from db.Model would give you this initializer (or something very similiar to it) for free.